### ISSUE TITLE
Sandbox Settings Tweaks

### ISSUE THREAD

**Issue by pieetrr** [Alziibun/Perdition#58]
For the wipe on the 26th i feel like we can try to make the gameplay more compelling by removing a lot of kiddy wheels from the server. I still don't think we are at the sweat spot yet as far as difficulty is concerned.

I propose we:
- Increase Zombie pop to starting 1.5 into 2.0 peak.
- Lower vehicle spawns
- Increase firearms rarity since the mod we added adds a lot of strong guns. Leave ammo as is.
- Increase loot respawn to 48 in game hours (0 unseen).
- Increase fuel consumption a bit more (esp since we have ways to craft fuel now)
- Decrease Hazmat Zombie spawn even more but make the chance of dropping a cure to 100%
- Further slight increase of hunger/thirst stats

If any of these prove to be too much we can always rollback the changes.
There's more stuff i want to change but i can't remember them rn so i'll add what i'm missing tomorrow.

**Comment by Alziibun** [comment]
why is this here and not on the discord

**Comment by pieetrr** [comment]
I said i would make an issue for sandbox settings, you nodded while saying mimimimimimi.

**Comment by Alziibun** [comment]
I thought it was gonna be something to do with editing the files but this requires none of that you silly

**Comment by TheQuaccer** [comment]
I recommend we keep our kiddy wheels on for our mods after until we are sure/balanced then we can take em off one by one rather than do it all at once. i recommend we make these changes one by one like every 3-5 irl days. for example zombie pop has been increased on 0.5 due to a huge migration from a nearby county. lower vehicle spawns after 6 days due as car use has plumeted. :3 this would make it more engaging

**Comment by Alziibun** [comment]
> I recommend we keep our kiddy wheels on for our mods after until we are sure/balanced then we can take em off one by one rather than do it all at once. i recommend we make these changes one by one like every 3-5 irl days. for example zombie pop has been increased on 0.5 due to a huge migration from a nearby county. lower vehicle spawns after 6 days due as car use has plumeted. :3 this would make it more engaging

tldr skill issue

**Comment by pieetrr** [comment]
> I recommend we keep our kiddy wheels on for our mods after until we are sure/balanced then we can take em off one by one rather than do it all at once. i recommend we make these changes one by one like every 3-5 irl days. for example zombie pop has been increased on 0.5 due to a huge migration from a nearby county. lower vehicle spawns after 6 days due as car use has plumeted. :3 this would make it more engaging

Quacc relax nothing will happen to you if things get rough for a week before we go back on a change. People need some challenge to play this game and currently our server doesnt provide that at all. 

> I thought it was gonna be something to do with editing the files but this requires none of that you silly

Hale asked me to make it an issue too. I think its not a bad place to talk about it.

**Comment by seishedaye** [comment]
I think all of pietrr's changes are good ideas, and that it wouldn't be awful to do them all at the same time.

**Comment by seishedaye** [comment]
However, maybe not the 100% zombie cure drop. I think just making hazmat zombies rarer would prevent someone like me from suddenly having like 20 cures she doesn't even use
